ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml

```diff
@@ -0,0 +1,145 @@
+base_images:
+  metrics-linter:
+    name: prom-metrics-linter
+    namespace: ci
+    tag: v0.0.2
```

Comment:
Was this ever updated sometime?

Reply:
Looks like it was: https://quay.io/repository/kubevirt/prom-metrics-linter?tab=tags&tag=latest

But it is out of the scope of this PR, and probably our team. I can open an issue.

Reply:
Please do, thanks!

Reply:
Issues are disabled on this repository.

@assafad , you were the author of the PR that added this linter. Do you think it should be updated?